### DISCUSSION TITLE
How to handle document ownership in a corporate environment

### DISCUSSION THREAD

**Discussion by turbo2ltr** [outline/outline#5796] [General] [answered]
I'm just getting started with outline to see if it is right for our company.  I tried deploying via docker than that was a disaster. Never had so much difficulty getting a docker image to work so I finally gave up and used the free trial.

In any case, my question is about ownership. Our intent is to use Outline for an internal knowledgebase.  If someone creates a page, and then they leave the company (using Microsoft SSO) and the document was not set to be editable how is that document then managed?  I don't see a way to change owners.  Am I missing something?

**Comment by tommoor** [comment, marked as answer]
Permissions are based at a collection level, not at documents. So if someone creates a page and leaves the workspace others with access to the collection will still be able to edit and manage it.

If the document is created in a collection where only the user that left has admin rights then it would not be possible to retrieve that in the community edition. The licensed editions + cloud include collection management for admins to change permissions on any collection, which would be a way to restore the content in this scenario.

**Comment by routerino** [comment]
> Permissions are based at a collection level, not at documents. So if someone creates a page and leaves others with access to the collection will still be able to edit and manage it.
> 
> If the document is created in a collection where only the user that left has admin rights then it would not be possible to retrieve that in the community edition. The licensed editions + cloud include collection management for admins to change permissions on any collection, which would be a way to restore the content in this scenario.

I am surprised that new collections would not force allow site admins to have access (which would also prevent potential orphaning issues). Was this a privacy consideration? It's odd from an admin perspective, for example admins in M365 have access to all e-mail in a tenancy, with the expectation that admin accounts are also appropriately monitored and restricted.

**Reply by tommoor** [reply]
> Was this a privacy consideration?

Yes, absolutely. Admins can give themselves access with the event logs that accompany that but they can't just arbitrarily look through private collections.

**Comment by routerino** [comment]
> > Was this a privacy consideration?
> 
> Yes, absolutely. Admins can give themselves access with the event logs that accompany that but they can't just arbitrarily look through private collections.

Does that include self hosted editions? I mean I can get saying "nah that's enterprise stuff", as long as that's the position (and a good thing [to add here](https://www.getoutline.com/pricing)). However [meshcentral](https://github.com/Ylianst/MeshCentral) for example solves this by having an optional environment variable to force all groups to be visible to the admin (so you basically start up in "recovery mode").

**Reply by tommoor** [reply]
The functionality for admin management of private collections is available on licensed editions + cloud as I mentioned in the top answer – it doesn't exist in the community edition.

Currently this is bundled under "Additional security controls" on the pricing page, I know more details are needed there 😓
